xml/System.Text.RegularExpressions/Regex.xml

```diff
@@ -332,8 +332,8 @@
 ## Examples  
  The following example illustrates how to use this constructor to instantiate a regular expression that matches any word that begins with the letters "a" or "t".  
   
- [!code-csharp[System.Text.RegularExpressions.Regex.Constructors#1](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.Text.RegularExpressions.Regex.Constructors/cs/constructors1.cs#1)]
- [!code-vb[System.Text.RegularExpressions.Regex.Constructors#1](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.Text.RegularExpressions.Regex.Constructors/vb/constructors1.vb#1)]  
+ :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_CLR_System/system.Text.RegularExpressions.Regex.Constructors/cs/constructors1.cs" interactive="try-dotnet" id="Snippet1":::
```

Comment:
There is also `id="Snippet1"` above... will this conflict?

Reply:
Yes, it's ok @eerhardt. it's on a different file. 

Feel free to ping me or @BillWagner when you see these interactive PRs.